account_financial_report_qweb/static/src/css/report.css

```diff
@@ -65,6 +65,7 @@
 .act_as_cell.amount {
     word-wrap:normal;
     text-align:right;
+    padding: 2px;
```

Comment:
Why this extra padding? This makes bigger full table.

Reply:
w/ padding 
![image](https://user-images.githubusercontent.com/45663766/51234832-98afbc80-1976-11e9-904a-68b6930304ee.png)

without 
![image](https://user-images.githubusercontent.com/45663766/51234920-ce54a580-1976-11e9-8129-d5a62a64176f.png)

maybe decrease it 1px would be optimal variant

Reply:
Well, why wasting that precious pixels in not so wide screens? I don't see any advantage.

Reply:
it probably not about wide, cos there a lot of free space and padding left, right have no influence on columns width,
 maybe only  on mobile devices, but for mobile there problems and without  padding 
and looks fine for screens from 1024px 
![image](https://user-images.githubusercontent.com/45663766/51248787-eb4ea000-1999-11e9-976a-90bfd599ba94.png)

it effects on height of report  but easier to read, open some item

Reply:
I prefer to remove all the paddings.

Reply:
done